client/src/components/JobCard.vue

```diff
@@ -116,6 +120,21 @@
         return classes;
       },
 
+      jobSectorName() {
+        return this.job.project.customer.sector.name;
+      },
+
+      showCountryLogo() {
+        if (this.job.project.customer.sector) {
+          return countries.indexOf(this.job.project.customer.sector.name) !== -1;
+        }
+        return false;
+      },
+
+      addPaddingToTitle() {
+        return this.showCountryLogo && this.job.activity.title.length < 20;
+      },
```

Comment:
Don't center job title and take whole width if more than 20 characters. (Arbitrary number...)

Reply:
a bit curious this feature. I would expect it.
As the jobboard is a MVP, let's give a try!